src/sim_recon/recon.py

```diff
@@ -326,16 +402,21 @@ def run_reconstructions(
 
                     async_results: list[AsyncResult] = []
                     for wavelength, processing_info in processing_info_dict.items():
+                        processing_info.status = ProcessingStatus.PENDING
```

Comment:
I think this is better inside the `reconstruct_...` function

Reply:
Because that's on a different process, that wouldn't get updated until the callback, at which point it is already updated to be `ProcessingStatus.COMPLETE`

Reply:
I did have a separate `ProcessingStatus.RUNNING` option that I removed when I realised the object wasn't being shared